inventory/byo/hosts.ose.example

```diff
@@ -167,6 +167,14 @@ openshift_release=v3.6
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_host: example.com:443
+registry_auth_user: some_user
+registry_auth_token: 'my-token'
```

Comment:
Just to ensure commonality with most of our variables what about `openshift_registry_auth_host` `openshift_registry_auth_user` and `openshift_registry_auth_password` instead? Password over token because it's more generic, for our use case where it's an openshift registry it happens to be a token but elsewhere it may just be a password.